docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -514,7 +523,42 @@ TIP: If you see errors such as `java.lang.IllegalArgumentException: Can not set
 [[loadbalanced-webclient]]
 === Spring WebFlux WebClient as a Load Balancer Client
 
-`WebClient` can be configured to use the `LoadBalancerClient`. `LoadBalancerExchangeFilterFunction` is auto-configured if `spring-webflux` is on the classpath. The following example shows how to configure a `WebClient` to use load balancer:
+[[webflux-with-reactive-loadbalancer]]
+==== Spring WebFlux WebClient with Reactive Load Balancer
```

Comment:
It seems to make sense that we should make the reactive load balancer the default and allow them to optionally configure it to use a non reactive version if they want to.  Is there any reason why we wouldnt do that?

Reply:
If they add the `spring-cloud-loadbalancer` dependency (which I will be probably turning into a starter for M3), they will have the reactive one by default; but if somebody just has the previously used `commons` dependency in the classpath, it will just continue working as previously if they upgrade to new commons version (this is to avoid backward incompatibility); wdyt?

Reply:
I guess its the case where `org.springframework.cloud:spring-cloud-loadbalancer` is not on the classpath.  I think that should be reactive by default.  I understand it might be a change in behavior but is it noticeable at all?  I don't see any negatives to it being reactive by default or causing any problems.

Reply:
Updated the docs.